lib/pgsync.rb

```diff
@@ -452,6 +455,18 @@ def primary_key(conn, table, schema)
       row && row["attname"]
     end
 
+    def typical_primary_key(conn, table, schema)
+      log "Fallback to typical primary key for #{schema}.#{table}"
+      query = <<-SQL
+        SELECT column_name
+        FROM information_schema.columns
+        WHERE table_schema = $1 AND table_name = $2 AND column_name IN ('id', 'code');
```

Comment:
Maybe we can use some query to get the primary key of a table?
Something like https://wiki.postgresql.org/wiki/Retrieve_primary_key_columns

To not hard-code possible primary key column names

Reply:
@st4lk I wish it was possible but primary constraints will available only on https://github.com/ergeon/erg-dbsync/blob/master/sync.sh#L40 step not on table data copy where `self.primary_key` called. Yesterday i try to modify such query to get "primary like" fields but it can not distinguish between `id`, and `user_id` without primary constraint and getting first of them is the only option.

```
ergid2=# SELECT a.attname, format_type(a.atttypid, a.atttypmod) AS data_type
FROM   pg_index i
JOIN   pg_attribute a ON a.attrelid = i.indrelid
                     AND a.attnum = ANY(i.indkey)
WHERE  i.indrelid = 'apps_role_users'::regclass
AND    i.indisprimary;
 attname | data_type 
---------+-----------
(0 rows)
```

Reply:
ok